### PR TITLE
Added all the topics as per issue 

### PR DESCRIPTION
**Pull request by Rishabhraghwendra18**
issue: #6 
Topics added:
- [x] Enhanced ER Model
- [x] Minimization of ER Diagram
- [x] ER Model: Generalization, Specialization and Aggregation
- [x] Recursive Relationships
- [x] Impedance Mismatch

Note: Not added ER Model topic as it was already added

### CODE REVIEW

File: ERmodel/ER Model: Generalization, Specialization and Aggregation/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# ER Model: Generalization, Specialization and Aggregation
```
Comment (Kr-Shanu) [inline ERmodel/ER Model: Generalization, Specialization and Aggregation/README.md:4]: Please mention your name and date of creation.

File: ERmodel/ER Model: Generalization, Specialization and Aggregation/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+![Aggregation](https://media.geeksforgeeks.org/wp-content/uploads/aggregation.png)
+
+### Representing aggregation via schema –
+To represent aggregation, create a schema containing:
+1. primary key of the aggregated relationship
+2. primary key of the associated entity set
+3. descriptive attribute, if exists.
+
+Source: geeksforgeeks.org
```
Comment (Kr-Shanu) [inline ERmodel/ER Model: Generalization, Specialization and Aggregation/README.md]: Please add the source link if possible.

**Comment by Kr-Shanu** [comment]
@Rishabhraghwendra18 Please mention your name and date of creation on top of every file.
Also try to mention the link of the source at last if possible.
Do add an introduction page : What is ER model ?

**Comment by Kr-Shanu** [comment]
It looks great @Rishabhraghwendra18

**Comment by Rishabhraghwendra18** [comment]
@Kr-Shanu thanks

**Pull request merged**